### DISCUSSION TITLE
Theme Picker

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#9824] [General] [answered]
Helix comes with a lot of different themes but there is no way to apply a theme without opening the `https://github.com/helix-editor/helix/tree/master/runtime/themes` url on browser. So, if a theme picker is added then it will be really fast to find the preferred theme.
And also there must be a way to check the toml file of the preferred theme.

**Comment by pascalkuthe** [comment, marked as answer]
You can use `:theme` which will suggest a list of themes and you can select them with tab. The theme is previewed right away. 

I have no idea what you are talking about opening an url. If helix is installed correctly all themes are installed automatically
